subjects/java/checkpoints/almost-palindrom/README.md

```diff
@@ -0,0 +1,39 @@
+## Palindrome
+
+### Instructions
+
+In a file named `AlmostPalindrome.java` write a function `isAlmostPalindrome` that returns true if the String in parameter is almost palindrome.
+A word is considered "almost a palindrome" if removing one letter from the word makes it a palindrome.
```

Comment:
Is the check case insensitive or cases counts? I would provide some example with capital letters or add some info in the subject to avoid confusion

Reply:
i changed the test to clarify that